.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deployment
+
+on:
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+  workflow_dispatch:
+    branch:
+      description: 'The branch to deploy'
+      default: 'main'
```

Comment:
Why would we ever manually deploy a branch other than `main`?

Reply:
It gives us the option pilot branches on a live URL before the PR process or for e2e testing in the future. This GH workflow can be extended to be configured to trigger by other events (like automated scheduled/periodic testing).